debian/rules

```diff
@@ -67,7 +67,7 @@ clean:
 	  ltmain.sh missing
 	rm -f configure Makefile.in man/Makefile.in src/Makefile.in
 	rm -f src/acconfig.h.in
-	rm -f debian/ceph-base.ceph.init debian/radosgw.init debian/ceph.logrotate debian/radosgw.logrotate
+	rm -f debian/ceph-base.ceph.init debian/radosgw.init debian/ceph-common.logrotate
```

Comment:
@tchaikov When a current jewel user upgrades to a new version containing this change, is this the line that cleans up the old files? I wonder if we should retain `debian/ceph.logrotate` here.